lib/apiservers/engine/backends/backends.go

```diff
@@ -112,7 +112,13 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 
 	serviceOptions := registry.ServiceOptions{}
 	for _, r := range insecureRegs {
-		insecureRegistries = append(insecureRegistries, r.Path)
+		/*
```

Comment:
Nitpick - we should use Go's idiomatic way of (non-package level) multi-line comments. We should stick to `//` for this.

https://golang.org/doc/effective_go.html#commentary
> Comments do not need extra formatting such as banners of stars.